CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.9.0 (2015-02-06)
+- Support for more network devices.
+- Support for more partition types.
```

Comment:
Can we be specific here instead of more? I guess we support btrfs in addition to existing "ext4" support?